common/persistence/src/main/resources/db/migration/V5__updateDiagnosisKeyTableForFederation.sql

```diff
@@ -1,5 +1,8 @@
 -- Migrate current values -> origin_country = DE
 
+ALTER TABLE diagnosis_key
+    ADD consent_to_federation BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
where do we store for which countries a key is valid? I'm not 100% sure, but maybe the customer want to ask people to give their consent per country for their keys...